files/brews/nginx.rb

```diff
@@ -32,21 +39,26 @@ def passenger_config_args
   end
 
   def install
+    pcre = Formula["pcre"]
+    openssl = Formula["openssl"]
+    cc_opt = "-I#{pcre.include} -I#{openssl.include}"
+    ld_opt = "-L#{pcre.lib} -L#{openssl.lib}"
+
     args = ["--prefix=#{prefix}",
             "--with-http_ssl_module",
             "--with-pcre",
             "--with-ipv6",
-            "--with-cc-opt='-I#{HOMEBREW_PREFIX}/include'",
-            "--with-ld-opt='-L#{HOMEBREW_PREFIX}/lib'",
+            "--with-cc-opt=#{cc_opt}",
+            "--with-ld-opt=#{ld_opt}",
             "--conf-path=/opt/boxen/config/nginx/nginx.conf",
             "--pid-path=/opt/boxen/data/nginx/nginx.pid",
             "--lock-path=/opt/boxen/data/nginx/nginx.lock"]
 
-    args << passenger_config_args if build.with? "passenger"
-    args << "--with-http_dav_module" if build.with? "webdav"
+    args << passenger_config_args if ARGV.include? '--with-passenger'
+    args << "--with-http_dav_module" if ARGV.include? '--with-webdav'
+    args << "--with-http_gzip_static_module" if ARGV.include? '--with-gzip-static'
+    args << "--with-http_v2_module" if ARGV.include? "--with-http2"
     args << "--with-debug" if build.with? "debug"
```

Comment:
This is the only reason that we are on a private fork. We want this debug option that isn't in the boxen upstream module. Might try to get a patch into the upstream eventually. But for now keeping a fork is fine!